ovs/extensions/healthcheck/volumedriver/volumedriver_health_check.py

```diff
@@ -258,19 +258,19 @@ def check_for_halted_volumes(logger):
                     except (ClusterNotReachableException, RuntimeError) as ex:
                         logger.failure(
                             "Seems like the Volumedriver {0} is not running.".format(vp.name, ex.message),
-                            'halted_{0}'.format(vp.name))
+                            'halted_volumedriver_{0}'.format(vp.name))
```

Comment:
When introducing a new parameter, you should also introduce a success so CHECKMK does not freak out